includes/class-newspack-popups-inserter.php

```diff
@@ -309,15 +309,18 @@ function ( $popup ) use ( $shortcoded_popups_ids ) {
 					$inline_popup['is_inserted'] = true;
 				}
 			}
-			$block_content = render_block( $block );
-			$output       .= $block_content;
+
+			// Render blocks from the block group.
+			foreach ( $block_group as $block ) {
+				$block_content = render_block( $block );
```

Comment:
```suggestion
				$block_content = serialize_block( $block );
```

We should return the block serialized instead of rendered to not interfere with other filters.

Reply:
Done in 8a543475c5fd1004387ba9f582e0a48ba3d558bd